pkgs/servers/tigerbeetle/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, zig
+}:
+stdenv.mkDerivation rec {
+  pname = "tigerbeetle";
+  version = "unstable-2022-09-30";
+
+  src = fetchFromGitHub {
+    owner = "tigerbeetledb";
+    repo = pname;
+    rev = "2023-02-13-weekly";
+    sha256 = "sha256-OcVLFnzW140Drx/jQ6p3eW1ZDGXrRSM6eQ47L+Ub54A=";
+  };
+
+  buildInputs = [ zig ];
+
+  buildPhase = ''
+    export HOME="$(mktemp -d)"
+    zig build -Dcpu=baseline -Drelease-safe
+  '';
```

Comment:
The phase definition looks reasonable. There is a convention to allod users to plug arbitrary code before and after the phase by defining their own `preBuild` / `postBuild` hooks. To do that each phase usually defines pre/post hook. I suggest the following definition:

```
  buildPhase = ''
    runHook preBuild

    export HOME="$(mktemp -d)"
    zig build -Dcpu=baseline -Drelease-safe

    runHook postBuild
  '';
```

You almost always see these `runHook` calls in `*Phase` functions.